packages/common/src/operations.graphql

```diff
@@ -92,6 +92,7 @@ query invoice($id: String!) {
             itemName
             numberOfPacks
             packSize
+            note
```

Comment:
off-road fix for the mock server note not working correctly

Reply:
thanks - only just noticed this one